.github/workflows/update.yml

```diff
@@ -0,0 +1,15 @@
+name: "Update"
+on:
+  repository_dispatch:
```

Comment:
Why do we need this one?

Reply:
Allows triggering the workflow from a github app, not really required. Let me remove it.